app/views/templates/base.php

```diff
@@ -121,11 +121,12 @@
 
   <script src="/assets/jquery/jquery.min.js?v=<?= VERSION ?>"></script>
   <script src="/assets/clipboard.js/clipboard.js-built.js?v=<?= VERSION ?>"></script>
-
-<?php foreach ($javascript_include as $js_file):?>
-  <script src="/js/<?= $js_file . $cache_bust ?>"></script>
-<?php endforeach?>
-
+<?php
+foreach ($javascript_include as $js_file) {
+    $prefix = ($js_file[0] == '/') ? '' : '/js/';
+    print '  <script src="' . $prefix . $js_file . $cache_bust . '"></script>' . "\n";
```

Comment:
Should we just update the existing includes instead? There are only 5 or 6 of them